Loop/View Controllers/SettingsTableViewController.swift

```diff
@@ -261,7 +261,7 @@ final class SettingsTableViewController: UITableViewController, IdentifiableClas
                     configCell.detailTextLabel?.text = SettingsTableViewCell.TapToSetString
                 }
             case .correctionRangeOverrides:
-                configCell.textLabel?.text = NSLocalizedString("Temporary Correction Ranges", comment: "The title text for the correction range overrides")
+                configCell.textLabel?.text = NSLocalizedString("Temporary Ranges", comment: "The title text for the correction range overrides")
```

Comment:
Is this officially the consistent title we should use now?  If so, consider changing `TherapySetting.swift` and using it here.

Reply:
Done!